packages/components/datetime/src/DateTime.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { CommonProps } from '@contentful/f36-core';
+
+import dayjs, { extend } from 'dayjs';
+import utcPlugin from 'dayjs/plugin/utc';
+extend(utcPlugin);
+
+export interface DateTimeProps extends CommonProps {
+  /**
+   * The date that will be displayed. It accepts a JS Date, an ISO8601 Timestamp string, or Unix Epoch Milliseconds number
+   */
+  date: Date | string | number;
+  /**
+   * The format in which the date will be presented
+   *
+   * @default 'full'
+   **/
+  format?: 'full' | 'time' | 'weekday' | 'day';
+  testId?: string;
```

Comment:
both testId and className are available in CommonProps already.

Reply:
ahhhh I guess another thing to update in the plop template

I will remove it